crypto/rsa/rsa_lib.c

```diff
@@ -185,6 +186,13 @@ int RSA_up_ref(RSA *r)
     return i > 1 ? 1 : 0;
 }
 
+#ifndef FIPS_MODULE
```

Comment:
I dont think we do the ifdef elsewhere for other key types??

Reply:
Fixed